### PR TITLE
Remove negative margins

### PR DESCRIPTION
**Pull request by hjheath**

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-259-pr-onfido-sdk-ui-onfido.surge.sh

File: src/components/Error/style.css [hunk 1]
```diff
@@ -4,8 +4,7 @@
   border-radius: 6px;
   text-align: left;
   color: #FFFFFF;
-  margin-top: -1rem;
-  margin-bottom: 27px;
+  margin-bottom: 16px;
```
Comment (stefaniacardenas) [inline src/components/Error/style.css]: Now you can remove `margin-top: 0` within the media query on line 12

**Comment by rfreitas** [comment]
Travis automatic deployment: https://259-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-259-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://259-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**